### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by Fiona-Waters**
Resolves https://github.com/project-codeflare/codeflare-operator/issues/195
Adding PR template for easier reviewing/verification of PRs.

### CODE REVIEW

**Review by z103cb — COMMENTED** [review]
LGTM!

File: .github/pull_request_template.md
Comment (Fiona-Waters) [reply .github/pull_request_template.md:17]: Yes I will update the last option to `Testing is not required` Thanks Alex!

**Review by accorvin — APPROVED** [review]

**Review by anishasthana — APPROVED** [review]

**Comment by accorvin** [comment]
@Fiona-Waters I've just given the `codeflare-developers` GitHub team maintainer permissions on this repo, so you should be able to merge this once you're happy with it.

**Pull request merged**
